### ISSUE TITLE
Problems with spaces in paths on Windows

### ISSUE THREAD

**Issue by nwoltman** [amilajack/eslint-plugin-flowtype-errors#81]
1: If my project is located at a path with a space above the project folder:

`C:\Users\me\project folders\project`

this plugin fails to start the Flow server and ESLint outputs `exec: internal error` ~~whenever Flow encounters a type error~~ several times (I can't find a correlation between the number of times that it is output and either the number of Flow errors or the number of files with the `@flow` flag).

2: If I have a file inside a folder with a space inside my project:

`C:\Users\me\projectFolders\project\js\space folder\fileWithTypeErrors.js`

this plugin fails to start the Flow server if run with the [`grunt-eslint`](https://github.com/sindresorhus/grunt-eslint) task or [`SublimeLinter-contrib-eslint`](https://github.com/roadhump/SublimeLinter-eslint). It does start the Flow server if run directly with the  command `node_modules/.bin/eslint .`, but in all cases, it doesn't report any Flow errors.

eslint-plugin-flowtype-errors: v3.2.0
flow-bin: v0.45.0
Windows 10

**Comment by amilajack** [comment]
Can you please show the entire stack trace

**Comment by nwoltman** [comment]
There is no stack trace. The `eslint` command completes successfully and `exec: internal error` is the only thing that gets written to the command line. Using eslint's `--debug` flag didn't yield anything useful.

Also, I updated the issue with more details.

**Comment by jdmota** [comment]
Maybe it is because we are doing

```js
new shell.ShellString(stdin)).exec([
   `${getFlowBin()}`,
   'check-contents --json --root',
   `${root} ${filepath}`
].join(' '), { silent: true })
```

So the arguments end up being

```
flow 
check-contents
--json
--root
C:\Users\me\projectFolders\project\js\space
folder
C:\Users\me\projectFolders\project\js\space
folder\fileWithTypeErrors.js
```

**Comment by amilajack** [comment]
@nwoltman Just published `3.2.1-0`. Would be awesome if you could test this out

**Comment by nwoltman** [comment]
Yes that fixed it. Thank you :)

**Comment by amilajack** [comment]
Great! Just published `3.2.1`

**Comment by amilajack** [comment]
@nwoltman also you should thank @jdmota. He PR'd the fix (as always)

**Comment by nwoltman** [comment]
Oh nice. Thanks @jdmota!

**Comment by jdmota** [comment]
You're welcome!
